Docs/docs/worlds/udon/networking/network-components.md

```diff
@@ -12,7 +12,17 @@ Special properties you can *get* from Networking:
 
 **IsClogged** - returns true if there is too much data trying to get out. You can use this to hold off some operations or adjust your logic.
 
-**IsInstanceOwner** - returns true if the Local Player is the one who created the instance. False when in Build & Test and Unity Playmode.
+**IsInstanceOwner** - returns true if the Local Player is the one who created the instance. False when in Build & Test and Unity Playmode. This value is controllable in Clientsim. Here is a table showing when this will be true
+| Instance Type | Will be true for |
+|---------------|------------------|
+| Public        | Nobody           |
+| Friends+      | Instance Creator |
+| Friends       | Instance Creator |
+| Invite+       | Instance Creator |
+| Invite        | Instance Creator |
+| Group+        | Nobody           |
+| Group         | Nobody           |
+| Group Public  | Nobody           |
```

Comment:
I would suggest rephrasing the last two sentences and making the table more compact.
```suggestion
**IsInstanceOwner** - returns true if the Local Player is the one who created the instance. False when in Build & Test and Unity Playmode. This value is controllable in ClientSim.

| Instance Type | Value of IsInstanceOwner |
| ------------- | ---------------- |
| Invite/Invite+<br/>Friends/Friends+ | `true` for the instance creator, otherwise `false`. |
| Group/Group+/Group Public<br/>Public | Always `false`.           |
```